packages/rocketchat-lib/server/models/Subscriptions.coffee

```diff
@@ -312,7 +312,6 @@ class ModelSubscriptions extends RocketChat.models._Base
 		update =
 			$set:
 				alert: true
-				open: true
```

Comment:
setAlertForRoomIdExcludingUserId is only called in one place and there was no expectation of the side effect of updating the room 'open' property.